plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogAccessOperations.java

```diff
@@ -181,7 +181,7 @@ public void testCreateTableAsSelect()
                             .add(GET_DATABASE)
                             .add(GET_DATABASE)
                             .add(CREATE_TABLE)
-                            .addCopies(GET_TABLE, 6)
+                            .addCopies(GET_TABLE, 5)
```

Comment:
nice!

do we know which change introduced this improvement?

Reply:
https://github.com/trinodb/trino/pull/16557 reduced the access count.

Reply:
thanks @ebyhr for the link.
thanks @electrum for the improvement.